packages/e2e-tests/specs/multi-block-selection.test.js

```diff
@@ -72,7 +74,11 @@ describe( 'Multi-block selection', () => {
 
 		// Select all via double shortcut.
 		await page.click( firstBlockSelector );
-		await pressKeyWithModifier( 'primary', 'a' );
+		// NOTE: `__unstableSelectAll` is used for cross-platform compatibility
+		// alternative to Cmd+A. The second issuance of the key combination is
+		// handled internerally by the block editor's KeyboardShortcuts utility
+		// and is not subject to the same buggy emulation.
+		await __unstableSelectAll();
```

Comment:
This one was a bit awkward to test, because it truly exists to ensure that Cmd+A twice has the correct behavior, so I opted to keep as close as possible to the original implementation to avoid using `selectAllBlocks`.